app/models/pseud.rb

```diff
@@ -112,6 +112,7 @@ class Pseud < ApplicationRecord
 
   scope :alphabetical, -> { order(:name) }
   scope :starting_with, lambda {|letter| where('SUBSTR(name,1,1) = ?', letter)}
+  scope :for_profile, -> { order(is_default: :desc).merge(alphabetical).limit(ArchiveConfig.ITEMS_PER_PAGE) }
```

Comment:
I'm a little iffy on the name of the scope, since it's not _just_ for the profile, but also the dashboard. I'm having trouble coming up with something better, though. `short_list` was my initial thought, but if we ever increase `ITEMS_PER_PAGE` to something like `100`, is that really short? I guess maybe, compared to the full list. `first_page` to go with the whole per page thing? 

Anyone else have better suggestions?

Reply:
`:alphabetical_preview`?

Reply:
This is the last outstanding thing, I'll let everyone percolate some last minute ideas on what we should name the bike shed while I sleep. ;-D

Reply:
@sarken's `abbreviated_list` won!